packages/web/src/components/ReportIssue.tsx

```diff
@@ -38,7 +38,7 @@ const useStyles = makeStyles(() =>
 
 const ReportIssue = () => {
   const { resourceId } = useParams();
-  const resource = useResource(resourceId || "");
+  const resource = useResources([resourceId || ""]).resources?.[0];
```

Comment:
```suggestion
  const resource = useResources([resourceId]).resources?.[0];
```
Missed this the first go around, but this route only renders when `resourceId` is defined in the URL 😅

Reply:
Ahh thought it would be nullable, didn't realize `resourceId` is `any` here... wonder if there's a more typesafe way of handling params with `react-router-dom` 🤔

Reply:
fd1e70477f611250b33a782525e2b3cc4189a1b5